src/main/java/sirius/web/templates/NamedRowProcessor.java

```diff
@@ -28,5 +27,5 @@
      * @param lineNumber the line number which is currently being processed.
      * @param row        the data of the row / line to process
      */
-    void handleRow(int lineNumber, Map<String, Value> row);
+    void handleRow(int lineNumber, ListMultimap<String, Value> row);
```

Comment:
but retrieving a single value for a column requires a complex lookup like  l = get(X); if (!l.isEmpty()) { val = l.get(0); }else{val = Value.EMPTY; }

which could be hidden in a getValue method. the 2nd getValues simply forwards to map.get ...